marketplace-and-integrations/packages/authorized-services.md

```diff
@@ -81,6 +81,15 @@ Below you can learn more about how to use the Umbraco Authorized Services packag
 
 The package should be installed into your Umbraco solution from [NuGet](https://www.nuget.org/packages/Umbraco.AuthorizedServices).
 
+### App Creation
+
+Services supported by the package will often offer an OAuth authentication and authorization flow against an "app" that the developer will need to create. This will make available information, including for example a "client ID" and "client secret", that will need to be applied in configuration.
```

Comment:
```suggestion
Services supported by the package will often offer an OAuth authentication and authorization flow against an "app" that the developer will need to create. This will make available information, including for example a "**client ID**" and "**client secret**", that will need to be applied in configuration.
```